### DISCUSSION TITLE
issue in OIDC configuration documentation

### DISCUSSION THREAD

**Discussion by danymat** [Syslifters/sysreptor#350] [General]
Hello, there is a small issue in the documents for OIDC configuration:

https://docs.sysreptor.com/users/oidc-setup/
https://docs.sysreptor.com/users/oidc-keycloak/
https://docs.sysreptor.com/users/oidc-generic/

The part

![image](https://github.com/user-attachments/assets/e57e8616-e082-4b55-b0d3-7b74ef461307)

Should contain the upper opening and closing brackets `{}`, such as:

```
OIDC_AUTHLIB_OAUTH_CLIENTS='{"<your provider name>": {"label": "<human readable provider name>",...}''
```

**Comment by MWedl** [comment]
Thanks for noticing and reporting the error. We will update the documentation

**Reply by MWedl** [reply]
We updated the documentation in https://github.com/Syslifters/sysreptor/releases/tag/2024.91
